modules/nf-core/parabricks/haplotypecaller/tests/main.nf.test

```diff
@@ -0,0 +1,198 @@
+nextflow_process {
+
+    name "Test Process PARABRICKS_HAPLOTYPECALLER"
+    script "../main.nf"
+    process "PARABRICKS_HAPLOTYPECALLER"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "parabricks"
+    tag "parabricks/haplotypecaller"
+
+    test("homo_sapiens - bam - no_intervals") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                            [ id:'test' ],
+                            [ file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/bam/test2.paired_end.recalibrated.sorted.bam', checkIfExists: true) ]
+                ]
+                input[1] = [
+                            [ id:'index' ],
+                            []
```

Comment:
i think you can also leave those completely empty
```suggestion
                input[1] = [
```

Reply:
Why? I have followed the structure of the pytest